src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksWebServiceCallsServiceImpl.java

```diff
@@ -198,18 +190,10 @@ private URI buildPaymentWorksNewVendorUpdateStatusURI() {
     }
     
     private Response buildJsonResponse(URI uri, String jsonString) {
-        Client client = null;
-        Response response = null;
-        try {
-            ClientConfig clientConfig = new ClientConfig();
-            client = ClientBuilder.newClient(clientConfig);
-            Invocation request = buildJsonClientRequest(client, uri, jsonString);
-            response = request.invoke();
-            response.bufferEntity();
-            return response;
-        } finally {
-            CURestClientUtils.closeQuietly(client);
-        }
+        Invocation request = buildJsonClientRequest(getClient(MultiPartFeature.class), uri, jsonString);
```

Comment:
This PaymentWorks service has a mix of `getClient()` and `getClient(MultiPartFeature.class)` calls. To ensure that the cached Client always has the MultiPartFeature registered, please update the code so that `getClient(MultiPartFeature.class)` will always be invoked accordingly.

To make the setup much simpler, you could have this class override the `getClient()` method so that it calls `getClient(MultiPartFeature.class)` explicitly. That way, the rest of the code can call the simpler `getClient()` method and still register the appropriate class.